static/app/views/settings/project/projectOwnership/ownershipRulesTable.spec.tsx

```diff
@@ -37,6 +37,27 @@ describe('OwnershipRulesTable', () => {
     expect(screen.getByText(user1.name)).toBeInTheDocument();
   });
 
+  it('should filter codeowners rules without actor names', () => {
+    const rules: ParsedOwnershipRule[] = [
+      {
+        matcher: {pattern: 'pattern', type: 'path'},
+        // @ts-expect-error this seems to only happen when adding a codeowners file
```

Comment:
This is a little unclear to me - how does it only happening for codeowners files relate to whatever TS is mad about?

Reply:
the type doesn't say it should ever be undefined and i wasn't successful at getting it to reproduce, so i'm telling typescript to ignore it instead of changing the type